openassessment/templates/openassessmentblock/edit/oa_edit_peer_assessment.html

```diff
@@ -37,6 +38,58 @@
                     </div>
                     <p class="setting-help">{% trans "Specify the number of learners that each response must be assessed by. Valid numbers are 0 to 99."%}</p>
                 </li>
+                <li class="field comp-setting-entry">
+                    <div class="wrapper-comp-setting">
+                        <label for="peer_assessment_enable_flexible_grading" class="setting-label">{% trans "Enable Flexible Peer Grade Averaging" %}</label>
+                        <select id="peer_assessment_enable_flexible_grading" class="input setting-input">
+                            <option value="0">{% trans "False" %}</option>
+                            <option value="1" {% if assessments.peer_assessment.enable_flexible_grading %}selected="true"{% endif %}>{% trans "True" %}</option>
+                        </select>
+                    </div>
+                    <p class="setting-help">{% trans "Learners who have received a minimum of {floor-30% of required peer-reviews configuration value} who have waited longer than 7 days will be given their grade instead of having to wait longer for more peer reviews" %}</p>
+                </li>
+                <li class="field comp-setting-entry">
```

Comment:
This probably doesn't belong here.
I think you're looking for the code that adds the toggle for average peer grading (above).